.github/workflows/verify.yml

```diff
@@ -25,70 +25,13 @@ on:
       - '*'
 
 jobs:
-  test:
-    runs-on: ${{ matrix.os }}
-    timeout-minutes: 40
-
-    services:
-      postgres:
-        image: postgres:9.6
-        ports: ["5432:5432"]
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-
-    strategy:
-      fail-fast: true
-      matrix:
-        ruby:
-          - '2.7'
-          - '3.0'
-          - '3.1'
-          - '3.2'
-        os:
-          - ubuntu-22.04
-          - ubuntu-latest
-        exclude:
-          - { os: ubuntu-latest, ruby: '2.7' }
-          - { os: ubuntu-latest, ruby: '3.0' }
-
-    env:
-      RAILS_ENV: test
-
-    name: ${{ matrix.os }} - Ruby ${{ matrix.ruby }}
-    steps:
-      - name: Install system dependencies
-        run: sudo apt-get install libpcap-dev graphviz
-
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: '${{ matrix.ruby }}'
-          bundler-cache: true
-
-      - name: Test
-        run: |
-          cp spec/dummy/config/database.yml.github_actions spec/dummy/config/database.yml
-          bundle exec rake --version
-          bundle exec rake db:create db:migrate
-
-          # Disabling this check because it is proving unreliable
-          # git diff --exit-code spec/dummy/db/structure.sql
-          bundle exec rake spec
-          bundle exec rake yard
-
-      - name: Upload coverage report
-        uses: actions/upload-artifact@v4
-        with:
-          name: coverage-${{ matrix.os }}-${{ matrix.ruby }}
-          path: |
-            coverage/
-          retention-days: 1
+  build:
+    uses: cgranleese-r7/metasploit-framework/.github/workflows/shared_gem_verify_rails.yml@add-gem-verify-shared-pipeline
+    with:
+      dependencies: '["libpcap-dev", "graphviz"]'
+      test_commands: |
+        cp spec/dummy/config/database.yml.github_actions spec/dummy/config/database.yml;
```

Comment:
Why the trailing `;` ? 👀

Reply:
Oh, that is a hangover when I was passing them in a different manner. Will remove 👍